macrosynergy/management/check_availability.py

```diff
@@ -3,130 +3,136 @@
 import matplotlib.pyplot as plt
 import seaborn as sns
 from typing import List, Union, Tuple
+import time
 
-from macrosynergy.management.simulate_quantamental_data import make_qdf
-from macrosynergy.management.shape_dfs import reduce_df
+from Simulate_Quantamental import make_qdf
```

Comment:
The line that is failing + lower cases for modules (files) ie. `simulate_quantamental`